docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+      remote_mount: nfsshare
+      local_mount: /home
+      fs_type: nfs
+
+  - id: appsfs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /apps
+      fs_type: nfs
+
+  - id: bucket-software
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_software)
+      local_mount: /user_provided_software
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs
+
+  - id: bucket-input
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_inputs)
+      local_mount: /data_input
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs,allow_other
+
+  - id: bucket-output
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_outputs)
+      local_mount: /data_output
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs,allow_other,dir_mode=0777,file_mode=766
+
+  ### Virtual Desktop ###
+
+  - id: desktop
+    source: ./community/modules/remote-desktop/chrome-remote-desktop
+    use:
+    - network1
+    - homefs
+    - appsfs
+    - bucket-input
+    - bucket-output
+    - bucket-software
+    settings:
+      name_prefix: chrome-remote-desktop
+      install_nvidia_driver: true
+      startup_script: |
+        sudo apt-get update
+        sudo apt-get -y upgrade
+        find /user_provided_software -name vmd-1.9.*.bin.LINUXAMD64*.tar.gz -exec tar xvzf  '{}' -C . \;
+        cd vmd-1.9.*/
+        ./configure
+        cd src/
+        sudo make install
+        cd ../..
+
+  ### Cluster ###
+
+  - id: debug_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 4
+      machine_type: n2-standard-2
+
+  - id: debug_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use:
+    - network1
+    - debug_node_group
+    - homefs
+    - appsfs
+    - bucket-input
+    - bucket-output
+    settings:
+      partition_name: debug
+      enable_placement: false
+      is_default: true
+
+  - id: compute_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 20
+
+  - id: compute_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use:
+    - network1
+    - compute_node_group
+    - homefs
+    - appsfs
+    - bucket-input
+    - bucket-output
+    settings:
+      partition_name: compute
+
+  - id: slurm_controller
+    source: community/modules/scheduler/schedmd-slurm-gcp-v5-controller
+    use:
+    - network1
+    - debug_partition
+    - compute_partition
+    - homefs
+    - appsfs
+    - bucket-input
+    - bucket-output
+    settings:
+      disable_controller_public_ips: false
```

Comment:
Here you enable public IP addresses for the controller and login nodes, but not for the compute nodes. Why?

Reply:
I believe this was a bad artifact from some testing. I have removed and will validate.